newdoc/newdoc/newdoc.py

```diff
@@ -92,7 +80,7 @@ def get_config():
 
         for k in options.keys():
             # The configparser library is different in Python 2 and 3.
-            # This si the only 2/3-compatible way I've found for optional keys.
+            # This is the only 2/3-compatible way I've found for optional keys.
             try:
                 options[k] = config.get("newdoc", k)
             except cp.NoOptionError:
```

Comment:
Maybe you want to somehow resolve this too? The comment seems to indicate you could do this "better". But then again, the code already works, so it does not matter much.

Reply:
You're absolutely right. It could be improved, but there's no reason to do it since the current code works.